lib/MooX/Options/Descriptive/Usage.pm

```diff
@@ -134,6 +134,10 @@ sub option_text {
     my @message;
     my $lf = _get_line_fold();
     for my $opt(@$getopt_options) {
+        if ($opt->{desc} eq 'spacer') {
```

Comment:
this line applies when $opt is text like it probably would be when someone used usage_top or usage_bottom